code/modules/events/swarmer.dm

```diff
@@ -1,10 +1,9 @@
 /datum/round_event_control/spawn_swarmer
 	name = "Spawn Swarmer Beacon"
 	typepath = /datum/round_event/spawn_swarmer
-	weight = 8
+	weight = 10
 	max_occurrences = 1 //Only once okay fam
-	earliest_start = 30 MINUTES
```

Comment:
?

Reply:
Normalizing base midround antagonist spawn rates as they get balanced.  I didn't do this for swarmers when I initially reworked them so I decided to do it here.